recipes/recipes_emscripten/r-matrix/recipe.yaml

```diff
@@ -0,0 +1,54 @@
+context:
+  name: r-matrix
+  version: 1.7-0
+  r_base_version: 4.4.1
+
+package:
+  name: ${{ name }}
+  version: ${{ version|replace('-', '_') }}
+
+source:
+  url:
+    - https://cran.r-project.org/src/contrib/Matrix_${{ version }}.tar.gz
+    - https://cloud.r-project.org/src/contrib/Matrix_${{ version }}.tar.gz
+  sha256: fb97bba0df370222eb4f7e2da2e94dd01053b5e054b1c51829ff9a6efc08ad37
+
+build:
+  number: 0
+  script:
+    - $R CMD INSTALL $R_ARGS --no-byte-compile .
+    - rm $PREFIX/lib/R/library/grDevices/libs/cairo.so
+    - rm $PREFIX/lib/R/library/tcltk/libs/tcltk.so
+    # The activation script of cross-r-base copies cairo and tcltk from
+    # the build environment.
+
+requirements:
+  build:
+  - cross-r-base >= ${{ r_base_version }}
+  - r-lattice
+  - ${{ compiler('c') }}
+  host:
+  - r-base == ${{ r_base_version }}
+  - r-lattice
+  - libblas>=3.12
+  - liblapack>=3.12
+  - libflang                  # for FortranRuntime
+  run:
+  - r-base == ${{ r_base_version }}
+  - r-lattice
+
+tests:
+- package_contents:
+    lib:
+    - R/library/Matrix/libs/Matrix.so
+
+about:
+  homepage: http://Matrix.R-forge.R-project.org/
+  license: GPL-2.0-or-later
+  license_family: GPL3
+  summary: A rich hierarchy of sparse and dense matrix classes, including general, symmetric, triangular, and diagonal matrices with numeric, logical, or pattern entries. Efficient methods for operating on such matrices, often wrapping the 'BLAS', 'LAPACK', and 'SuiteSparse' libraries.
```

Comment:
Could you split this summary into multiple lines please?

Reply:
Ahh yes, haven't formatted that well. Will do !

Reply:
Addressed